template/_gitignore

```diff
@@ -21,6 +21,7 @@ DerivedData
 *.ipa
 *.xcuserstate
 project.xcworkspace
```

Comment:
I guess we can remove this line too since we're changing this file :D

Reply:
@satya164 @kelset done! 🙂